### ISSUE TITLE
2.12.0 java client needs 2.8.6 gson

### ISSUE THREAD

**Issue by space88man** [OpenVidu/openvidu#424]
**Describe the bug**
js-java tutorial with 2.12.0 java client uses 2.8.5 version of gson. The `parseString()` method is not supported resulting in `java.lang.NoSuchMethodError`.

**Expected behavior**
js-java/ runs (as with 2.11.0)

**Wrong current behavior**
No such method exception: parseString() (as we had 2.8.5 gson)

**OpenVidu tutorial where to replicate the error**
js-java
**OpenVidu deployment info**
Ubuntu 18.04

```
Logging in | {user, pass}={"user":"publisher2","pass":"pass"}                                                                    
Getting a token from OpenVidu Server | {sessionName}={"sessionName":"Session 0"}                                                        
New session Session 0                                                                                                               
2020-04-07 06:42:40.076 ERROR 1235 --- [nio-5000-exec-3] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in cont
ext with path [] threw exception [Handler dispatch failed; nested exception is java.lang.NoSuchMethodError: com.google.gson.JsonParser.parseString(Ljava/lang
/String;)Lcom/google/gson/JsonElement;] with root cause                                                  
                                                                                                                                   
java.lang.NoSuchMethodError: com.google.gson.JsonParser.parseString(Ljava/lang/String;)Lcom/google/gson/JsonElement;
        at io.openvidu.java.client.Session.httpResponseToJson(Session.java:499) ~[openvidu-java-client-2.12.0.jar:na]
        at io.openvidu.java.client.Session.getSessionIdHttp(Session.java:481) ~[openvidu-java-client-2.12.0.jar:na]                                          
        at io.openvidu.java.client.Session.<init>(Session.java:58) ~[openvidu-java-client-2.12.0.jar:na]
        at io.openvidu.java.client.OpenVidu.createSession(OpenVidu.java:131) ~[openvidu-java-client-2.12.0.jar:na]
        at io.openvidu.js.java.SessionController.getToken(SessionController.java:113) ~[classes/:na]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_242]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_242]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_242]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_242]    
```

**Additional Information**

If I force the version of gson to 2.8.6 in `openvidu-tutorials/openvidu-js-java/pom.xml`
it works as expected.

**Comment by space88man** [comment]
Dependency tree:
```
mvn dependency:tree

[INFO] \- io.openvidu:openvidu-java-client:jar:2.12.0:compile
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.8:compile
[INFO]    |  +- org.apache.httpcomponents:httpcore:jar:4.4.11:compile
[INFO]    |  \- commons-codec:commons-codec:jar:1.11:compile
[INFO]    +- com.google.code.gson:gson:jar:2.8.5:compile
[INFO]    \- org.slf4j:slf4j-api:jar:1.7.26:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.717 s
[INFO] Finished at: 2020-04-07T06:58:12Z
[INFO] ------------------------------------------------------------------------
```

**Comment by space88man** [comment]
```
git checkout v2.12.0
cd openvidu-js-java/
mvn dependency:tree

[INFO] \- io.openvidu:openvidu-java-client:jar:2.12.0:compile
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.8:compile
[INFO]    |  +- org.apache.httpcomponents:httpcore:jar:4.4.11:compile
[INFO]    |  \- commons-codec:commons-codec:jar:1.11:compile
[INFO]    +- com.google.code.gson:gson:jar:2.8.5:compile
[INFO]    \- org.slf4j:slf4j-api:jar:1.7.26:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.624 s
[INFO] Finished at: 2020-04-07T07:05:20Z
[INFO] ------------------------------------------------------------------------
```

**Comment by pabloFuente** [comment]
Yes, for hat reason a bug fix 2.12.1 was released for openvidu-java-client. It solves this issue.
